src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -38,6 +38,55 @@ private static IEnumerable<string> DumpTypeRefs(MetadataReader[] readers)
             }
         }
 
+        [Fact]
+        public void Diagnostics_Nullable()
```

Comment:
>Diagnostics_Nullable [](http://example.com/codeflow?start=20&length=20)

Nullable analysis for attribute applications is not handled by MethodCompiler. Are warnings still reported for them?

Reply:
Can you give a code example?

Reply:
> Can you give a code example?

```
#nullable enable

[MyTest(null)]
class MyTestAttribute : System.Attribute
{
    public MyTestAttribute(string x) {}
}
```

Reply:
Thanks!

Reply:
Seems like we can't easily skip the analysis in this case as we don't have emit context available. I think that's ok, since skipping the analysis is an optimization and does not need to be perfect. Attribute and default parameter values are trivial, so their analysis shouldn't be expensive.